### ISSUE TITLE
A null character is being inserted into formatted attributes, preventing texts from being saved.

### ISSUE THREAD

**Issue by zdavis** [ManifoldScholar/manifold#3092]
@scryptmouse — I have a manifest you can use to reproduce this error. When I give it to you, you can reproduce with:

```
rails manifold:project:ingest["some-project-id","path/to/manifest.yaml"]
```

Ingesting the manifest is leading to this error:

```
ActiveRecord::StatementInvalid: PG::UntranslatableCharacter: ERROR:  unsupported Unicode escape sequence
DETAIL:  \u0000 cannot be converted to text.
CONTEXT:  JSON data, line 1: ...__rights_holder":{"formatted":"","plaintext":""},...
```

I'm not convinced that it's something in the manifest. My suspicion is that something in the text compiler is leading to a nil value for a metadata property, and that somewhere in formatted attributes we need to cast nils as strings. I might be wrong about this — would appreciate it if you could dig in deeper.

**Comment by zdavis** [comment]
@bot bug

**Comment by zdavis** [comment]
Thank you for taking the time to report this bug. We've reviewed the issue and agree that this should be fixed. We’re adding a “planned” label to indicate that we consider this bug as part of our current work plan.

_This was an automated message, but please don't hesitate to reply. Our team watches these issues closely and will respond as soon as we're able to!_

**Comment by zdavis** [comment]
Please open the PR for this against v6.x

**Comment by zdavis** [comment]
This was resolved in v6.0
